report-deployment-status-to-slack/action.yml

```diff
@@ -60,7 +60,10 @@ runs:
       id: setup
       shell: bash
       run: |
-        if [[ -n "${{ inputs.db_migration_detected }}" && ${{ inputs.db_migration_detected }} == true ]]; then
+        if [[ -z "${{ inputs.db_migration_detected }}" ]]; then
+          ${{ inputs.db_migration_detected }}=false
+        fi
```

Comment:
It looks like this addition is making one of the jobs fail.

Reply:
Yes, I'm revamping the action to accommodate the new edge test-case